adage/analyze/tests.py

```diff
@@ -709,18 +723,16 @@ def test_edge_union(self):
         id2 = Gene.objects.last().id   # The last gene
 
         # The number of edges in the union of the first and last genes
-        # should be: num_gene1 + num_gene2 - 1, "-1" is due to double
-        # counting of the edge between the first and last genes.
+        # should be equal to num_gene1 * num_gene2.
```

Comment:
This works out, but only because the test data set is a fully-connected network of 100 genes. What I think we're looking for is more subtle, as described above.